### ISSUE TITLE
No access to cache files through .htaccess rule?

### ISSUE THREAD

**Issue by robo-wg** [getkirby/staticache#23]
Is it possible that this line of the .htaccess blocks access to the cached files?

```
RewriteRule ^site/(.*) index.php [L]
```

I am currently solving it this way. Is this the right way?

```
RewriteCond %{REQUEST_URI} !/cache/(.*)/pages/ 
RewriteRule ^site/(.*) index.php [L]
```

**Comment by lukasbestle** [comment]
Good point. The current Apache example in the plugin docs cannot work because the `[L]` flag tells Apache to apply all rules again to the new rewritten URL (resulting in the block by the site rule).

I currently don't have an Apache test server at hand. Could you try if replacing `[L]` with `[END]` in the Staticache rules works? You can remove the `RewriteCond` you added to the `site` rule.

```apache
RewriteCond %{DOCUMENT_ROOT}/site/cache/%{SERVER_NAME}/pages/%{REQUEST_URI}/index.html -f [NC]
RewriteRule ^(.*) %{DOCUMENT_ROOT}/site/cache/%{SERVER_NAME}/pages/%{REQUEST_URI}/index.html [END]

RewriteCond %{DOCUMENT_ROOT}/site/cache/%{SERVER_NAME}/pages/%{REQUEST_URI} -f [NC]
RewriteRule ^(.*) %{DOCUMENT_ROOT}/site/cache/%{SERVER_NAME}/pages/%{REQUEST_URI} [END]
```

**Comment by robo-wg** [comment]
@lukasbestle This is of course the more elegant solution and works perfectly for me. It would be great, of course, if this were also changed in the README. I think that would avoid some frustration with this great plugin :-)

**Comment by lukasbestle** [comment]
Yes, I'll certainly add this to the README. Just wanted to confirm first whether it solves the problem.

**Comment by robo-wg** [comment]
@lukasbestle So in my attempt with the configuration of the hosting provider where I tried it, it worked.
